ECMAScript/react-native/src/components/QuilttConnector.tsx

```diff
@@ -35,6 +48,39 @@ export const QuilttConnector = ({
   const { session } = useQuilttSession()
   oauthRedirectUrl = encodeURIComponent(oauthRedirectUrl)
   const connectorUrl = `https://${connectorId}.quiltt.app/?mode=webview&oauth_redirect_url=${oauthRedirectUrl}&agent=react-native-${version}`
+  const [preFlightCheck, setPreFlightCheck] = useState<PreFlightCheck>({ checked: false })
+
+  const checkConnectorUrl = useCallback(
+    async (retryCount = 0): Promise<PreFlightCheck> => {
+      const response = await fetch(connectorUrl)
+      if (!response.ok) {
+        // errorReporter.notify(
+        //   `Retry exhausted, failed to fetch connectorUrl: ${connectorUrl}, status: ${response.status}`
+        // )
+        console.error(`The URL ${connectorUrl} is not routable.`)
+        if (retryCount < 5) {
```

Comment:
Let's make this a CONST. Also I think 3 would be fine. I would additionally add a sleep thats like 50 ms; So

`Try -> Retry x3 -> Wait (count + 1 * 50) -> Error`
`Try -> Wait 50 -> Try -> Wait 100 -> Try -> Wait 150 -> Try -> Error`